rs/tests/ckbtc/ckbtc_minter_kyt.rs

```diff
@@ -318,29 +331,15 @@ pub fn test_kyt(env: TestEnv) {
         }) = retrieve_result
         {
             assert_eq!(error_code, 1);
-            assert_eq!(
-                error_message,
-                "Destination address is tainted, KYT check fee deducted: 0.00001001"
-            );
+            assert_eq!(error_message, "Destination address is tainted");
         } else {
             panic!("Expected to see a tainted destination address.")
         }
-        assert_burn_transaction(&ledger_agent, &logger, 4, &withdrawal_account, KYT_FEE).await;
 
-        upgrade_kyt(&mut kyt_canister, KytMode::AcceptAll).await;
-        let _ = minter_agent.distribute_kyt_fee().await;
+        upgrade_new_kyt(&mut new_kyt_canister, NewKytMode::AcceptAll).await;
 
-        assert_mint_transaction(
-            &ledger_agent,
-            &logger,
-            5,
-            &Account {
-                owner: agent_principal,
-                subaccount: None,
-            },
-            KYT_FEE,
-        )
-        .await;
+        // This should do nothing since retrieve_btc no longer charges KYT fee
```

Comment:
Should we mark this endpoint as deprecated?

Reply:
The KYT canister canister gets cycles for its work so that it is self-sustainable from the start. It doesn't need ckBTC.
There are no external providers anymore that need to pay a license fee, so the entire complexity of doing distributions can be removed.
Only the ckBTC minter accumulates ckBTC in fees, which it is going to sell in the future to ensure that it is itself self-sustainable.

Reply:
Created a ticket to remove distribute_kyt_fee (but not before we upgrade the ckBTC minter to using the new KYT canister) https://dfinity.atlassian.net/browse/XC-237